packages/lesswrong/components/ultraFeed/UltraFeedPostDialog.tsx

```diff
@@ -661,6 +673,10 @@ const UltraFeedPostDialog = ({
                     />
                   )}
                 </div>
+                {/* placeholders for side comments, reacts, and notes */}
+                <div />
+                <div />
+                <div />
```

Comment:
?

Reply:
used to match the grid layout of PostsPage.tsx and get good centering